src/sage/rings/function_field/drinfeld_modules/homset.py

```diff
@@ -20,13 +20,85 @@
 #                   http://www.gnu.org/licenses/
 # *****************************************************************************
 
+import operator
+
 from sage.categories.drinfeld_modules import DrinfeldModules
 from sage.categories.homset import Homset
+from sage.categories.action import Action
 from sage.misc.latex import latex
 from sage.rings.function_field.drinfeld_modules.morphism import DrinfeldModuleMorphism
 from sage.structure.parent import Parent
 
 
+class DrinfeldModuleMorphismAction(Action):
+    r"""
+    Action of the function ring on the homset of a Drinfeld module.
```

Comment:
I would suggest to be a bit more verbose and, for example, define the action and showcase one or two examples on why this class is useful.

Reply:
Done